### ISSUE TITLE
Running :NightfoxCompile failed

### ISSUE THREAD

**Issue by LC0117** [EdenEast/nightfox.nvim#133]
## Description

As described above.

## Behavior

**Expected behavior**
Action performed successfully.

**Actual Behavior**
```
E5108: Error executing lua .../packer/start/nightfox.nvim/lua/nightfox/lib/compile.lua:138: attempt to index local 'opts' (a nil value)
stack traceback:
        .../packer/start/nightfox.nvim/lua/nightfox/lib/compile.lua:138: in function 'compile'
        ...te/pack/packer/start/nightfox.nvim/lua/nightfox/init.lua:24: in function 'compile'
        [string ":lua"]:1: in main chunk
```

## Configuration

```lua
options = {
  dim_inactive = false,
  styles = {
    functions = 'italic'
  },
  inverse = {match_paren = true},
  transparent = false
}
```
**Reproduce**

1. Update to newest branch. 
2. Run `:NightfoxClean` and `:NightfoxCompile`

## Additional info / context
neovim version: Hombrew neovim 0.7 on macOS arm64 12.3

**Comment by LC0117** [comment]
### Thanks for your great plugin and quick respond!

However, there is still a issue with the inverse options, when it's on, I got message like this when execute `colorscheme {some}fox`

```
E5108: Error executing lua ...te/pack/packer/start/nightfox.nvim/lua/nightfox/main.lua:58: Vim(luafile):E5113: Error while calling lua chunk: ~/.cac
he/nvim/nightfox/duskfox_compiled.lua:188: invalid key: inverse
stack traceback:
        [C]: in function 'nvim_set_hl'
        ~/.cache/nvim/nightfox/duskfox_compiled.lua:188: in main chunk
        [C]: in function 'cmd'
        ...te/pack/packer/start/nightfox.nvim/lua/nightfox/main.lua:58: in function 'load'
        [string ":lua"]:18: in main chunk
stack traceback:
        [C]: in function 'cmd'
        ...te/pack/packer/start/nightfox.nvim/lua/nightfox/main.lua:58: in function 'load'
        [string ":lua"]:18: in main chunk
```

**Comment by EdenEast** [comment]
It looks like `nvim_set_hl` wants the key called `reverse` instead of `inverse`. That is resolved with PR #137.

**Comment by LC0117** [comment]
Thanks!!
It works really nice now!
